templates/instances/index.html

```diff
@@ -80,4 +89,70 @@ <h1 class="title is-1">{{ _("Join BookWyrm") }}</h1>
         </p>
     </div>
 </section>
-{% endblock %}
+<script>
+    // Get references to DOM elements
+    const cardsContainer = document.getElementById('cards');
+    const cards = Array.prototype.slice.call(document.querySelectorAll('.column.is-6'));
+    const sortBtn = document.getElementById('sort-btn');
+    const sortDropdown = document.getElementById('sort-options');
+    const sortOptions = document.querySelectorAll('#sort-options > span');
+
+    const sort = option => {
+        switch (option) {
+            case 'Users':
+                cards.sort((a, b) => {
+                    // Process HTML node text to get raw number and coerce string to num with +.
+                    // This sorts by descending value.
+                    const first = +a.querySelector('.users').innerText.split(' ')[0].replace(',', '');
```

Comment:
I think it be clearer here to add a `data-` attribute to the element that contains just the raw value, so that you don't have to parse out the comma in the number of the `Users` and `Version ...` text

Reply:
Agreed, will change